sql/catalyst/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Implicits.scala

```diff
@@ -62,4 +74,31 @@ object DataSourceV2Implicits {
       new CaseInsensitiveStringMap(options.asJava)
     }
   }
+
+  def convertPartitionIndentifers(
+      partSpec: TablePartitionSpec,
+      partSchema: StructType): InternalRow = {
```

Comment:
Why is this included with the implicits when it isn't an implicit class?

Reply:
Hm, a little ugly to me if it defined with implicits. I can change it if you think it's better with implicits.

Reply:
I don't think it needs to be implicit. I just don't think it belongs in the implicits class if it isn't an implicit. I think there is a util class you could include this in.

Reply:
Ah, I misunderstood. thanks.